tests/unittest/test_load_yaml.py

```diff
@@ -0,0 +1,32 @@
+
+# Generated by CodiumAI
+
+import pytest
+from pr_agent.algo.utils import load_yaml
+
+
+class TestLoadYaml:
+    #  Tests that load_yaml loads a valid YAML string
+    def test_load_valid_yaml(self):
+        yaml_str = 'name: John Smith\nage: 35'
+        expected_output = {'name': 'John Smith', 'age': 35}
+        assert load_yaml(yaml_str) == expected_output
```

Comment:
**Suggestion:** Consider using a fixture to setup and teardown your test data. This will make your tests cleaner and easier to maintain.
```suggestion
        @pytest.fixture
        def setup_data():
            return 'name: John Smith\nage: 35', {'name': 'John Smith', 'age': 35'}

        def test_load_valid_yaml(self, setup_data):
            yaml_str, expected_output = setup_data
            assert load_yaml(yaml_str) == expected_output
```